src/opera/template/relationship.py

```diff
@@ -47,58 +47,46 @@ def run_operation(self,
     def get_property(self, params):
         host, prop, *_ = params
 
-        if host == OperationHost.TARGET.value:
-            raise DataError("{} is not yet supported in opera.".format(OperationHost.HOST.value))
-        if host != OperationHost.SELF.value:
-            raise DataError(
-                "Property host should be set to '{}' which is the only valid value. This is needed to indicate that "
-                "the property is referenced locally from something in the node itself. "
-                "Was: {}".format(OperationHost.SELF.value, host)
-            )
-
-        # TODO(@tadeboro): Add support for nested property values once we
-        # have data type support.
-        if prop not in self.properties:
-            raise DataError("Template has no '{}' property".format(prop))
-
-        return self.properties[prop].eval(self, prop)
-
-    def get_attribute(self, params):
-        host, attr, *_ = params
+        if host == OperationHost.SELF.value:
+            # TODO(@tadeboro): Add support for nested property values.
```

Comment:
tadej!

Reply:
I've erased this tag as well and have reduced the opera burden on Tadej's shoulders.